src/config/audiences.js

```diff
@@ -28,7 +28,7 @@ export default {
     },
     staging: {
       clientId: '8qY2xJob1JAxhmVhIDLCNnGriTM9bct8',
```

Comment:
i **think** this needs to be updated too:
```suggestion
      clientId: 'qGzdTXcmB57zlTp86rYsivG9qEss1lbF',
```

Reply:
Hmmm, any idea on how I could check that?

Reply:
this should be right, i pulled this from the api itself: https://github.com/ndustrialio/contxt-api/blob/14c3199dbb082c81b041f48b9214fc7c9ce7fbc9/charts/contxt/values-staging.yaml#L59-L60

Reply:
to verify, we can just grab a JWT for that audience and try to make a request. the only wrinkle here is it looks like that client only exists in the staging auth0 tenant: https://github.com/ndustrialio/contxt-api/blob/14c3199dbb082c81b041f48b9214fc7c9ce7fbc9/charts/contxt/values-staging.yaml#L86-L87